src/play-item.js

```diff
@@ -0,0 +1,42 @@
+import {setup} from './auto-advance.js';
+
+/**
+ * Removes all remote text tracks from a player.
+ *
+ * @param  {Player} player
+ */
+const clearTracks = (player) => {
+  let tracks = player.remoteTextTracks() || [];
+
```

Comment:
The reason why `while(i--){}` was used previously is because `TextTrackList` is a `live` list. Does `forEach` cache the array's value?